docs/install/knative-with-operators.md

```diff
@@ -1,84 +1,74 @@
-weight: 10
+title: "Knative Operator installation"
+weight: 02
 type: "docs"
+showlandingtoc: "false"
 ---
 
-Knative provides an [operator](https://github.com/knative/operator) as a tool to install, configure and manage Knative. The Knative operator leverages custom objects
-in the cluster to define and manage the installed Knative software. This guide explains how to install and uninstall
-Knative using Knative operator.
+Knative provides a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to install, configure and manage Knative.
 
-## Before you begin
-
-Knative installation using the Operator requires the following:
-
-- A Kubernetes cluster v1.16 or newer, as well as a compatible kubectl. This guide assumes that you've already created
-a Kubernetes cluster. If you have only one node for your cluster, set CPUs to at least 6, Memory to at least 6.0 GB,
-Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set CPUs to at least 2, Memory to at least
-4.0 GB, Disk storage to at least 20 GB for each node.
-- The Kubernetes cluster must be able to access the internet, since the Knative operator downloads images online.
-- [Download and install Istio](./installing-istio.md).
-
-## Limitations of Knative Operator:
-
-Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
+**NOTE:** The Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
 for development or test purposes only.
 
-## Install Knative with the Knative Operator
+## Prerequisites
 
-You can find the release information of Knative Operator on the [Releases page](https://github.com/knative/operator/releases).
+- You have a cluster that uses Kubernetes v1.17 or newer.
+- You have installed the `kubectl` CLI.
+- If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
+- If you have multiple nodes in your cluster, for each node you will need at least 2 CPUs, 4 GB of memory, and 20 GB of disk storage.
+- Your Kubernetes cluster must have access to the internet, since the Knative Operator must download images.
+- You have installed [Istio](./installing-istio.md).
 
-### Installing the Knative Operator
+## Installing the latest release
 
-__From releases__:
+You can find the release information about the Knative Operator on the [Releases page](https://github.com/knative/operator/releases).
 
-Install the latest Knative operator with the following command:
+To install the latest stable Operator release, enter:
 
 ```
 kubectl apply -f {{< artifact org="knative" repo="operator" file="operator.yaml" >}}
 ```
 
-__From source code__:
+## Installing from source
 
-You can also install Knative Operator from source using `ko`.
+You can install the Knative Operator from the source code using the [ko](https://github.com/google/ko) build tool.
```

Comment:
Might be worth explaining why you would want to do this, or moving this to https://github.com/knative/operator/tree/master/DEVELOPMENT.md

@houshengbo , WDYT?

Reply:
This should not be in user land docs it should be in github dev/contrib md

Reply:
@evankanderson I can move this section into the dev part of operator.

Reply:
Removing this bit in follow up PR